Expert installation instructions do not list make as a dependency

> To install the `rasterio` you need `g++` and for `pypublicdecompwt` you need `make`. Should we add `make` to the [expert installation instructions](https://github.com/NRLMMD-GEOIPS/geoips/blob/9aafb0d1712eedcbbf551b6b069d40af484bd7ca/docs/source/starter/expert_installation.rst#id5)?

I think so !

_Originally posted by @mindyls in https://github.com/NRLMMD-GEOIPS/geoips/issues/452#issuecomment-2000000425_

# Expert installation instructions do not list make as a dependency

### Description
*  `pypublicdecompwt` is a dependency of geoips. `make` is a dependency of `pypublicdecompwt`. To install `pypublicdecompwt` you need `make`, but it's not listed as a dependency on the expert installation instructions. It needs added. Found when updating the Dockerfile. Re: #449 and #452

### Code to demonstrate issue

- Install without make

``` bash
gwynu@Gwyns-MacBook-Pro geoips % docker build -t geoips .                                
[+] Building 298.2s (19/19) FINISHED                                                                                                                                                                             docker:desktop-linux
 => [internal] load build definition from Dockerfile                                                                                                                                                                             0.0s
 => => transferring dockerfile: 1.71kB                                                                                                                                                                                           0.0s
 => [internal] load metadata for docker.io/library/python:3.10-slim-bullseye                                                                                                                                                     1.8s
 => [internal] load .dockerignore                                                                                                                                                                                                0.0s
 => => transferring context: 72B                                                                                                                                                                                                 0.0s
 => [ 1/15] FROM docker.io/library/python:3.10-slim-bullseye@sha256:e89739e9b11753ace337116c9086c4ba780c0c3daff118a7c680783cf7afada1                                                                                             0.0s
 => [internal] load build context                                                                                                                                                                                                0.0s
 => => transferring context: 201.34kB                                                                                                                                                                                            0.0s
 => CACHED [ 2/15] RUN apt-get update && apt-get -y upgrade                                                                                                                                                                      0.0s
 => [ 3/15] RUN apt-get install -y wget git libopenblas-dev imagemagick g++ imagemagick                                                                                                                                         ```
...
``` bash
 => ERROR [15/15] RUN cd /app/geoips_packages/geoips     && pip install --no-cache . #".[doc,lint,test,debug]"                                                                                                                 147.5s
------                                                                                                                                                                                                                                
 > [15/15] RUN cd /app/geoips_packages/geoips     && pip install --no-cache . #".[doc,lint,test,debug]":                                                                                                                              
0.190 Defaulting to user installation because normal site-packages is not writeable                                                                                                                                                   
0.219 Processing /app/geoips_packages/geoips                                                                                                                                                                                          
0.221   Installing build dependencies: started                                                                                                                                                                                        
3.114   Installing build dependencies: finished with status 'done'                                                                                                                                                                    
3.115   Getting requirements to build wheel: started
3.197   Getting requirements to build wheel: finished with status 'done'
```
...
``` bash
80.36   Building wheel for pypublicdecompwt (pyproject.toml): started
80.44   Building wheel for pypublicdecompwt (pyproject.toml): finished with status 'error'
80.44   error: subprocess-exited-with-error
80.44   
80.44   × Building wheel for pypublicdecompwt (pyproject.toml) did not run successfully.
80.44   │ exit code: 1
80.44   ╰─> [63 lines of output]
80.44       running bdist_wheel
80.44       running build
80.44       running build_ext
80.44       CMake Error: CMake was unable to find a build program corresponding to "Unix Makefiles".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
80.44       CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
80.44       CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
```
...
``` bash
146.7 ERROR: Could not build wheels for pypublicdecompwt, which is required to install pyproject.toml-based projects
------
Dockerfile:48
--------------------
  47 |     WORKDIR ${GEOIPS_PACKAGES_DIR}/geoips
  48 | >>> RUN cd ${GEOIPS_PACKAGES_DIR}/geoips \
  49 | >>>     && pip install --no-cache . #".[doc,lint,test,debug]"
  50 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c cd ${GEOIPS_PACKAGES_DIR}/geoips     && pip install --no-cache . #\".[doc,lint,test,debug]\"" did not complete successfully: exit code: 1

View build details: docker-desktop://dashboard/build/desktop-linux/desktop-linux/ov416xu6ys869pzy4acq8o7wm

```

### Checklist for Completion
* [ ] Add `pypublicdecompwt` to dependencies on the expert installation instructions

--- a/docs/source/releases/v1_12_2a0.rst
+++ b/docs/source/releases/v1_12_2a0.rst
@@ -19,6 +19,7 @@ Version 1.12.2a0 (2024-02-21)
   * Ensure that log.interactive works from plugins when imported independently.
   * Make add_log_level a callable class to fix AttributeErrors.
   * Modify algorithm_interpolator_colormapper to only provide input xarray
+  * Re-write Dockerfile to restore functionality (installing geoips/passing tests)
 
 
 * Refactor
@@ -85,6 +86,21 @@ despite it no longer being used. It also updates the documentation accordingly.
     modified: setup/check_system_requirements.sh
     modified: docs/source/starter/expert_installation.rst
     modified: docs/source/starter/mac_installation.rst
+
+
+Add make to expert installation dependencies
+--------------------------------------------
+..
+  *From NRLMMD-GEOIPS/geoips#454: 2024-03-15, Add make to expert installation dependencies*
+
+
+`pypublicdecompwt` is a dependency of geoips. `make` is a dependency of `pypublicdecompwt`. 
+To install `pypublicdecompwt` you need `make`, but it's not listed as a dependency on 
+the expert installation instructions. It was added.
+
+::
+
+    modified: docs/source/starter/expert_installation.rst
 
 Refactor
 ========

--- a/docs/source/starter/expert_installation.rst
+++ b/docs/source/starter/expert_installation.rst
@@ -38,6 +38,7 @@ Required (**included in**
 * wget (Miniconda installation)
 * git >= 2.19.1 (git -C commands in complete installation)
 * openblas (required for scipy pip install)
+* make (required for pypublicdecompwt)
 * Python >= 3.9 (3.9 required for entry points)
 * Test data repos contained in $GEOIPS_TESTDATA_DIR
   (required for tests to pass)

